src/main/js/components/header/index.js

```diff
@@ -13,6 +13,10 @@ function init() {
     }
   });
 
+  window.addEventListener("computeOverflow", () => {
```

Comment:
should this be scoped that it is only computing the overlflow for the header?

e.g.

```suggestion
  window.addEventListener("computeHeaderOverflow", () => {
```